online-shop/src/app/product-details/product-details.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="navigation">
```

Comment:
make the names meaningful. This one does not represent a navigation component.
If you want to apply some style to the component itself you can use the :host in the scss file.
https://blog.angular-university.io/angular-host-context/ (look at :host)